src/software/thunderscope/common/proto_configuration_widget.py

```diff
@@ -36,13 +45,16 @@ def __init__(
         self.setLayout(layout)
 
         self.on_change_callback = on_change_callback
-        self.proto_to_configure = proto_to_configure
 
         # Create search query bar
         self.search_query = QLineEdit()
         self.search_query.textChanged.connect(self.__handle_search_query_changed)
         self.search_filter_threshold = search_filter_threshold
 
+        self.is_yellow = is_yellow
```

Comment:
The same parameter file should be shared between both colors, so independent of the color we are during a game at Robocup, we have the same parameters

Reply:
indeed, but we need this parameter for the sensor fusion config.